README.md

```diff
@@ -1,178 +1,136 @@
 # CloudWeGo-Kitex
+English | [中文](README_cn.md)
 
-Kitex[kaɪt'eks] 字节跳动内部的 Golang 微服务 RPC 框架，具有**高性能**、**强可扩展**的特点，在字节内部已广泛使用。如今越来越多的微服务选择使用 Golang，我们考虑将 Kitex 开源出来丰富开源社区。如果对微服务性能有要求，又希望定制扩展融入自己的治理体系，Kitex 会是一个不错的选择。
+Kitex [kaɪt'eks] is a **High-Performance** and **Strong-Extensibility** Golang RPC framework that can help developers build microservices. As more and more organizations are now choosing Golang, if you need high-performance microservices and want to do some customization, Kitex will be a good choice.
 
-## 框架特点
+## Basic Features
 
-- **高性能**
+- **High Performance**
 
-  使用自研的高性能网络库 [Netpoll](https://github.com/cloudwego/netpoll)，性能相较 go net 具有显著优势。
+Kitex integrates [Netpoll](https://github.com/cloudwego/netpoll), a high-performance network library, which offers significant performance advantage over [go net](https://pkg.go.dev/net).
 
-- **扩展性**
+- **Extensibility**
 
-  提供了较多的扩展接口以及默认扩展实现，使用者也可以根据需要自行定制扩展，具体见下面的框架扩展。
+Kitex provides more extension interfaces and default extension implementations. Developers can also customize extensions according to their own needs (please refer to the framework extension section below).
```

Comment:
Kitex provides many extension interfaces with default extension implementations. Developers can also customize extensions according to their own needs (please refer to the framework extension section below). => There are plentiful interfaces with default implementation for users to customize each aspect of Kitex. You can extend and inject them into Kitex to fulfill your needs (please refer to the framework extension section below).

Reply:
done